### PR TITLE
Fix #6 : Add ability to tell it is a site admin

### LINKED ISSUE #6: Ability to tell a site admin if this admin already had a different role
Currently if a site admin already had other roles, for example enrolled in a course. Then the site admin would not have a record in the table stating his site admin role.  If not other role, then its able to tell that is site admin.

### PR DESCRIPTION
**Pull request by yao9394**
* Additional row to tell it is a site admin when this admin already had a different role.

### CODE REVIEW

File: classes/table/accessaudit_table.php [hunk 1]
```diff
@@ -157,6 +159,14 @@ protected function get_sql_and_params($count = false) {
              LEFT JOIN {role} r ON ra.roleid = r.id";
         $sql .= " WHERE $where";
```
Comment (brendanheywood) [inline classes/table/accessaudit_table.php:166]: I think this WHERE should be on the outside and apply to both selects, add ( ) around the selects so its really clear what the UNION is doing

File: classes/table/accessaudit_table.php [hunk 2]
```diff
@@ -142,6 +142,8 @@ public function query_db($pagesize, $useinitialsbar = true) {
      * @return array containing sql to use and an array of params.
      */
     protected function get_sql_and_params($count = false) {
+        global $CFG;
+
         if ($count) {
             $select = "COUNT(1)";
         } else {
```
Comment (brendanheywood) [inline classes/table/accessaudit_table.php:153]: Split this $select into two parts, a $userselect, and a $roleselect, you can reuse the first in the both sql below,but only the $userselect can be used in the first one

File: classes/table/accessaudit_table.php (hunk 1 above)
Comment (yao9394) [reply classes/table/accessaudit_table.php:166]: Hi @brendanheywood  This is addressed in commit 4de144be78a96d392ba0af40772a179037e29d49

File: classes/table/accessaudit_table.php (hunk 2 above)
Comment (yao9394) [reply classes/table/accessaudit_table.php:153]: Hi @brendanheywood  This is fixed in commit 4de144be78a96d392ba0af40772a179037e29d49

**Pull request merged**